src/encoded/tests/data/master-inserts/user.json

```diff
@@ -298,7 +298,7 @@
     },
     {
         "email": "ozturkutku@gmail.com",
-        "uuid": "336b1174-8b79-4c39-a3ce-298be2cc1509",
+        "uuid": "1ade49a9-78e9-4d38-b127-92ac65598137",
```

Comment:
Is this the uuid from prod?

Reply:
Prod and wolf; don't know why it was different here, but it was causing master inserts loading to crash.

Reply:
Ok yes unifying with those envs is the right choice just wanted to be sure